platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/files/DataFileCollection.java

```diff
@@ -575,6 +552,18 @@ Set<Integer> getSetOfNewFileIndexes() {
         }
     }
 
+    @Override
+    public LongSummaryStatistics getFilesSizeStatistics() {
```

Comment:
Strictly speaking, this method lies. There may be up to 2 data files in this file collection, which are not completed yet, and therefore their sizes will not be included to the stats. My _guess_ is this is why the method is currently called `getAllCompletedFilesSizeStatistics()` rather than `getFilesSizeStatistics()`. At the very least, JavaDoc for this overridden method should be updated to mention that only completed files are included